OrbitLinuxTracing/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ cmake_minimum_required(VERSION 3.15)
 
 project(OrbitLinuxTracing)
 
-add_library(OrbitLinuxTracing STATIC)
+add_library(OrbitLinuxTracing STATIC include/OrbitLinuxTracing/TracingOptions.h)
```

Comment:
Not exactly the CMake person here, but shouldn't `include/OrbitLinuxTracing/TracingOptions.h` be added to `target_sources` below like the other public headers?

Reply:
Oh thanks for spotting this. That was autogen by Clion. I'll add it below.